specs/protocol/withdrawals.md

```diff
@@ -10,6 +29,8 @@
 [g-relayer]: ../glossary.md#withdrawals
 [g-execution-engine]: ../glossary.md#execution-engine
 
+## Overview
+
```

Comment:
There is a `<!-- toc -->` tag below this, can we remove it?

Reply:
Yes sorry I thought I got all of them